cmd/skopeo/copy.go

```diff
@@ -131,5 +146,14 @@ var copyCmd = cli.Command{
 			Value: "",
 			Usage: "`DIRECTORY` to use to store retrieved blobs (OCI layout destinations only)",
 		},
+		cli.StringFlag{
+			Name:  "format",
+			Value: "oci",
```

Comment:
The default behavior should be to use the existing autodetection logic.

Reply:
fixed